### DISCUSSION TITLE
Welcome to OpenCopilot Discussions!

### DISCUSSION THREAD

**Discussion by gharbat** [opencx-labs/copilot#74] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by therealdhrxv** [comment]
is there a discord server or a reddit sub for this community?

**Reply by krcpr007** [reply]
Discord: https://discord.gg/yjEgCgvefr
